src/rgw/driver/rados/rgw_zone.cc

```diff
@@ -1351,6 +1351,26 @@ int RGWZoneGroupPlacementTier::update_params(const JSONFormattable& config)
       retain_head_object = false;
     }
   }
+  if (config.exists("allow_read_through")) {
+    string s = config["allow_read_through"];
+    if (s == "true") {
+      allow_read_through = true;
+    } else {
+      allow_read_through = false;
+    }
+  }
+  if (config.exists("read_through_restore_days")) {
+    r = conf_to_uint64(config, "read_through_restore_days", &read_through_restore_days);
+    if (r < 0) {
+      read_through_restore_days = DEFAULT_READ_THROUGH_RESTORE_DAYS;
+    }
+  }
+  if (config.exists("cloud_restore_object_hold_period")) {
```

Comment:
As discussed this option is no longer needed. Please remove all the instances of this option.

Reply:
Please remove this option related changes